### ISSUE TITLE
L2 regularization

### ISSUE THREAD

**Issue by viplix3** [maiminh1996/YOLOv3-tensorflow#15]
In your script train.py at line #110 you are computing the loss like

    with tf.name_scope("Loss_and_Detect"):

        # Label

        y_predict = [Y1, Y2, Y3]

        # Calculate loss

        loss = compute_loss(scale_total, y_predict, anchors, len(classes_data), print_loss=False)

        # loss_print = compute_loss(scale_total, y_predict, anchors, len(classes_data), print_loss=False)

        tf.summary.scalar("Loss", loss)`

compute_loss funtion is computing the YOLO loss, I get that.
But I cannot find where are you (if you are) accounting the loss contibuted by L2 weights regularization as mentioned in the original paper.

**Comment by YunYang1994** [comment]
https://github.com/YunYang1994/tensorflow-yolov3

**Comment by jaelim** [comment]
@YunYang1994 Hi, So can you describe your codes, like if it's ready to be used for training?

**Comment by viplix3** [comment]
Solved it myself https://github.com/viplix3/YOLO
Closing.
